geth/params/testdata/config.rinkeby.json

```diff
@@ -21,6 +21,10 @@
     "LogFile": "geth.log",
     "LogLevel": "INFO",
     "LogToStderr": true,
+    "UpstreamConfig": {
+        "Enabled": false,
+        "URL": "https://mainnet.infura.io/nKmXgiFgc2KqtoQ8BCGJ"
```

Comment:
Yes it is for the test case, here we either use that for `Ropsten` if we need to change.